### PR TITLE
[Docs] Fix broken links

### PR DESCRIPTION
**Pull request by JuYeong0413**
This patch updates broken links in contributing guide.

Signed-off-by: Juyeong Lee <2jy22@naver.com>

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The submitter has the first responsibility of keeping the created PR clean and n
 A PR is required to meet the following criteria.
 * It has passed all the tests defined for TAOS-CI.
     - This includes unit tests and integration tests in various platforms and different static analysis tools.
-    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](CODE_OF_CONDUCT.md). You may need to refer to later section.
+    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](https://github.com/nnstreamer/nntrainer/blob/main/CODE_OF_CONDUCT.md). You may need to refer to later section.
```
Comment (myungjoo) [inline docs/contributing.md:43]: Same here. If ```.../CODE_OF_CONDUCT.md``` works, it'd be better.

**Review by zhoonit — APPROVED** [review]
LGTM :)

**Review by lhs8928 — APPROVED** [review]
LGTM

File: docs/contributing.md (hunk 1 above)
Comment (jijoongmoon) [reply docs/contributing.md:43]: I think just CODE_OF_CONDUCT.md could works. 
(You can see https://github.com/nnstreamer/nntrainer/edit/main/CONTRIBUTING.md#L54 )

Comment (JuYeong0413) [reply docs/contributing.md:43]:
```markdown
[Code of Conduct](CODE_OF_CONDUCT.md)
```
works in `github.io` but breaks `github.com` 😂
In the [pull request for nnstreamer docs](https://github.com/nnstreamer/nnstreamer/pull/3099), the links were changed to apply the direct link for now.

Maybe I should wait until the [discussion about broken links in the nnstreamer repository](https://github.com/nnstreamer/nnstreamer/issues/3100) ends?

Comment (jijoongmoon) [reply docs/contributing.md:43]: If it breaks, then it is ok for this. Thanks.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**